src/_locales/de/messages.json

```diff
@@ -0,0 +1,254 @@
+    "description": "Cookie Cleanup"
+  },
+
+  "cookieCleanupIgnoreOpenTabsText": {
+    "message": "Cookies aufräumen (offene Tabs ignorieren)",
+    "description": "Cookie Cleanup (Ignore Open Tabs)"
+  },
+
+  "clearCookiesForDomainText": {
+    "message": "Alle Cookies dieser Domain löschen",
+    "description": "Clear All Cookies For This Domain"
+  },
+
+  "activeModeSwitchText": {
+    "message": "Aktiver Modus",
+    "description": "Active Mode"
+  },
+
+  "switchToWhiteListText": {
+    "message": "Zur Whitelist hinzufügen?",
+    "description": "Add to WhiteList?"
+  },
+
+  "welcomeText": {
+    "message": "Willkommen",
+    "description": "Welcome"
+  },
+
+  "settingsText": {
+    "message": "Cookie-Einstellungen",
+    "description": "Cookie Settings"
+  },
+
+  "whiteListText": {
+    "message": "Cookie-Whitelist",
+    "description": "Cookie WhiteList"
+  },
+
+  "aboutText": {
+    "message": "Über",
+    "description": "About"
+  },
+
+  "welcomeMessage": {
+    "message": "Cookie AutoDelete hat während dieser Sitzung $sessionDeleted$ und insgesamt $totalDeleted$ Cookies gelöscht.",
+    "description": "Hi there! During this session, Cookie AutoDelete has deleted 123 cookies and in total 456 cookies.",
+    "placeholders": {
+      "sessionDeleted" : {
+        "content" : "$1",
+        "example" : "123"
+      },
+      "totalDeleted" : {
+        "content" : "$2",
+        "example" : "456"
+      }
+    }
+  },
+
+  "reviewLinkMessage": {
+    "message": "Danke für das Ausprobieren von Cookie AutoDelete. Falls es dir gefällt, verfasse bitte einen Erfahrungsbericht.",
+    "description": "Thanks for trying out Cookie AutoDelete. If you liked it, then please give a Review."
+  },
+
+  "releaseNotesText": {
+    "message": "Versionshinweise",
+    "description": "Release Notes"
+  },
+
+"activeModeText": {
+    "message": "Aktiven Modus einschalten? Verzögerung bis zum Aufräumen:",
+    "description": "Enable Active Mode? Delay Before Cleaning:"
+  },
+
+"minutesText": {
+    "message": "Minute(n)",
+    "description": "Minute(s)"
+  },
+
+"activeModeTooltipText": {
+    "message": "Wenn der aktive Modus eingeschaltet ist wird das Aufräumen der Cookies beim Schließen eines Tabs ausgeführt",
+    "description": "When active mode is enabled, cookie cleanup is triggered on tab close."
+  },
+
+"statLoggingText": {
+    "message": "Gesamtanzahl gelöschter Cookies protokollieren",
+    "description": "Log Total Number Of Cookies Deleted"
+  },
+
+"statLoggingTooltipText": {
+    "message": "Zählt die Anzahl der gelöschten Cookies, wird im Willkommensbildschirm angezeigt",
+    "description": "Counts the number of cookies deleted during this session and in total. This is shown in the Welcome Screen."
+  },
+
+"resetCounterText": {
+    "message": "Zähler zurücksetzen",
+    "description": "Reset Counter"
+  },
+
+"showNumberOfCookiesInIconText": {
+    "message": "Zeigt die Anzahl der Cookies einer Domain in der Symbolleiste",
```

Comment:
I added "in the toolbar" here, also "in the icons tooltip in the toolbar" in the next translation string. Otherwise the text doesn't add anything helpful for the user, in my opinion.